jest.config.js

```diff
@@ -8,6 +8,9 @@ module.exports = {
       }
     ]
   },
+  moduleNameMapper: {
+    npm_package: '<rootDir>/src/module.ts'
```

Comment:
What if simply mapping from `src` to make code portable and clear what alias means?